lib/serviceBusMessage.ts

```diff
@@ -943,6 +943,11 @@ export class ServiceBusMessage implements ReceivedMessage {
         condition: options.deadletterReason,
         description: options.deadLetterErrorDescription
       };
+    } else {
```

Comment:
This could work, but I think we might be able to define the method like this instead:

```typescript
async deadLetter(options: DeadLetterOptions = {deadletterReason:"", deadLetterErrorDescription:"" }): Promise<void> {
```

If we do that, then if the options property is not specified it will be the empty strings (or some other values that we decide on).

/cc @amarzavery @ramya-rao-a

Reply:
Result for both would be the same but your's is more succinct.